subsys/bluetooth/audio/unicast_client.c

```diff
@@ -227,11 +238,14 @@ static struct bt_iso_chan_ops unicast_client_iso_ops = {
 static void unicast_client_ep_init(struct bt_audio_ep *ep, uint16_t handle,
 				   uint8_t dir)
 {
+	struct bt_unicast_client_ep *client;
```

Comment:
suggest to use `client_ep` instead of `client`, as `client` would be more fitting for something broader than a single remote ASE

Reply:
This comment applies multiple places